packages/driver/package.json

```
@@ -52,7 +54,7 @@
     "moment": "2.24.0",
     "morgan": "1.9.1",
     "npm-install-version": "6.0.2",
-    "parse-domain": "2.3.4",
+    "parse-domain": "github:bahmutov/parse-domain#fb60bd4",
```

never seen this before, we usually do it like this:

```suggestion
    "parse-domain": "bahmutov/parse-domain#fb60bd4",
```

is there any difference?